src/test/ui/parser/macro-braces-dot-question.rs

```diff
@@ -0,0 +1,9 @@
+// check-pass
+
+use std::io::Write;
+
+fn main() -> Result<(), std::io::Error> {
```

Comment:
Some negative tests such as `vec![1, 2, 3].len()` still failing to build would be great.